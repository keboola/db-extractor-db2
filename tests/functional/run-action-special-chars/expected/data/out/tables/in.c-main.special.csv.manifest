{
    "destination": "in.c-main.special",
    "incremental": false,
    "metadata": [
        {
            "key": "KBC.name",
            "value": "special"
        },
        {
            "key": "KBC.sanitizedName",
            "value": "special"
        },
        {
            "key": "KBC.schema",
            "value": "SAMPLE"
        },
        {
            "key": "KBC.type",
            "value": "table"
        }
    ],
    "column_metadata": {
        "col1": [
            {
                "key": "KBC.datatype.type",
                "value": "VARCHAR"
            },
            {
                "key": "KBC.datatype.nullable",
                "value": true
            },
            {
                "key": "KBC.datatype.basetype",
                "value": "STRING"
            },
            {
                "key": "KBC.datatype.length",
                "value": "4096"
            },
            {
                "key": "KBC.datatype.default",
                "value": ""
            },
            {
                "key": "KBC.sourceName",
                "value": "col1"
            },
            {
                "key": "KBC.sanitizedName",
                "value": "col1"
            },
            {
                "key": "KBC.primaryKey",
                "value": false
            },
            {
                "key": "KBC.uniqueKey",
                "value": false
            },
            {
                "key": "KBC.ordinalPosition",
                "value": 1
            }
        ],
        "col2": [
            {
                "key": "KBC.datatype.type",
                "value": "VARCHAR"
            },
            {
                "key": "KBC.datatype.nullable",
                "value": true
            },
            {
                "key": "KBC.datatype.basetype",
                "value": "STRING"
            },
            {
                "key": "KBC.datatype.length",
                "value": "4096"
            },
            {
                "key": "KBC.datatype.default",
                "value": ""
            },
            {
                "key": "KBC.sourceName",
                "value": "col2"
            },
            {
                "key": "KBC.sanitizedName",
                "value": "col2"
            },
            {
                "key": "KBC.primaryKey",
                "value": false
            },
            {
                "key": "KBC.uniqueKey",
                "value": false
            },
            {
                "key": "KBC.ordinalPosition",
                "value": 2
            }
        ]
    },
    "columns": [
        "col1",
        "col2"
    ]
}
