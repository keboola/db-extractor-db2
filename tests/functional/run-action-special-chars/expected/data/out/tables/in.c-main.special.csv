"column with backslash \ inside","column with backslash and enclosure \"""
"column with enclosure "", and comma inside text","second column enclosure in text """
"column with \n \t \\","second col"
"columns with
new line","columns with       tab"
"first","something with

double new line"
"line with enclosure","second column"
"unicode characters","ľščťžýáíéúäôň"
"column with escaped escape character """""""" inside","column with multiple escaped escape characters """""""""" inside"
"column with escaped enclosure at end of string: test\""","\""column with escaped enclosure at beginning of string"
"column with multiple escaped enclosures: test \""example\"""" here","\""column with escaped enclosure and delimiter\"""
"column with multiple escape characters """""""""""" inside","column with escape character at end of string: test"""""
"column with unescaped enclosure at end of string: test""","""column with unescaped enclosure at beginning of string"
